src/databasetasks.cpp

```diff
@@ -71,11 +71,11 @@ void DatabaseTasks::runTask(const DatabaseTask& task)
 	bool success;
 	DBResult_ptr result;
 	if (task.store) {
-		result = db.storeQuery(task.query);
+		result = Database::storeQuery(task.query);
```

Comment:
Now I see it. It shouldn't have a singleton-like interface then, but rather be instantiated and referenced where needed. What is the advantage of having multiple connections? It could be a connection pool instead.